main.tf

```
@@ -106,6 +133,19 @@ resource "aws_iam_role" "default" {
   tags                 = module.this.tags
 }
 
+resource "aws_iam_role_policy_attachment" "ssm_core" {
+  count      = module.this.enabled ? local.instance_profile_count : 0
```

```suggestion
  count      = local.enabled ? local.instance_profile_count : 0
```

```
locals {
   enabled = module.this.enabled
```

Update it everywhere